src/lib/mongoBackend/mongoRegisterContext.cpp

```diff
@@ -71,7 +71,7 @@ HttpStatusCode mongoRegisterContext
 )
 {
   bool         reqSemTaken;
-  std::string  sPath = (servicePath == "")? DEFAULT_SERVICE_PATH_UPDATES : servicePath;
+  std::string  sPath = (servicePath == "")? SERVICE_PATH_ROOT : servicePath;
```

Comment:
Same here about the `// FIXME Px: this has to be fixed to work as in NGSIv2` mark.

Reply:
Issue number (to be included in the FIXME mark): telefonicaid#3078

Reply:
Fixed in 88d49e6